src/taskpane/components/ReferenceList.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  ITheme,
+  getTheme,
+  List,
+  mergeStyleSets,
+  getFocusStyle,
+  FocusZone,
+  FocusZoneDirection,
+  Checkbox,
+} from "@fluentui/react";
+import React from "react";
+
+const theme: ITheme = getTheme();
+const { palette, semanticColors, fonts } = theme;
+interface bib {
+  isSelected: boolean;
+  id: string;
+  author: string;
+  title: string;
+  journal?: string;
+  volume?: string;
+  number?: string;
+  pages?: string;
+  year?: string;
+  DOI?: string;
+  type?: string;
+  abstract?: string;
+  keywords?: string;
+  citationKey?: string;
+  other?: unknown;
+}
```

Comment:
I am not sure what is the best option for this (see Skype). You want it to be checkbox specific?

I'd suggest moving everything except `isSelected` to `data.ts` and export it as a type,

```
export type BibliographyEntry = {...
```
, give this interface a more meaningful name (if it is only intended for checkboxes) and extend that type here in this file 

```
interface bib extends BibliographyEntry {
  isSelected: boolean;
```

I believe this approach will make it easier to add types in function arguments.

In my other comments, I'll say `BibliographyEntry` but if you don't agree with the suggested changes just replace it with `bib`.